### DISCUSSION TITLE
登上绳网只为发初中好兄弟帅照？？？

### DISCUSSION THREAD

**Discussion by SMLTxuxT** [share121/inter-knot#2287] [其他]
![123](https://github.com/user-attachments/assets/b4818c9d-a038-4f3c-a0a8-e62f09f1c7aa)

**Comment by KUYUE404** [comment]
你吓到我了，赔偿200丁尼
![Uploading 1723712361011.gif…]()

**Comment by SMLTxuxT** [comment]
1纪念币，爱要不要qmq

**Comment by shenguojia123** [comment]
![358213405-abced010-63c6-411b-9d89-140a79958672](https://github.com/user-attachments/assets/734b0d5a-101a-4a3b-aa5a-25154c20c2a5)
